src/main/java/org/liliya/hotelapp/ui/Console.java

```diff
@@ -0,0 +1,119 @@
+package org.liliya.hotelapp.ui;
+
+import org.liliya.hotelapp.exception.ReservationException;
+import org.liliya.hotelapp.model.Apartment;
+import org.liliya.hotelapp.model.Client;
+import org.liliya.hotelapp.service.ApartmentService;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Scanner;
+
+public class Console {
+    private final ApartmentService apartmentService;
+    private final Scanner scanner;
+
+    public Console(ApartmentService apartmentService) {
+        this.apartmentService = apartmentService;
+        this.scanner = new Scanner(System.in);
+    }
+
+    public void start() {
+        while (true) {
```

Comment:
make some logic to exit safer from app
like 
boolean isWork = true;

while(isWork){
...
some exit logic 
isWork = false;

Reply:
fixed